foreign-chains/solana/sol-prim/src/consts.rs

```diff
@@ -34,11 +34,16 @@ pub const BPF_LOADER_UPGRADEABLE_ID: Address =
 pub const COMPUTE_BUDGET_PROGRAM: Address =
 	const_address("ComputeBudget111111111111111111111111111111");
 
-pub const MAX_TRANSACTION_LENGTH: usize = 1_232;
+pub const MAX_TRANSACTION_LENGTH: usize = 1_232usize;
 pub const MAX_COMPUTE_UNITS_PER_TRANSACTION: u32 = 1_400_000u32;
 pub const MICROLAMPORTS_PER_LAMPORT: u32 = 1_000_000u32;
 pub const LAMPORTS_PER_SIGNATURE: u64 = 5000u64;
 
-pub const NONCE_ACCOUNT_LENGTH: u64 = 80;
+pub const NONCE_ACCOUNT_LENGTH: u64 = 80u64;
 
 pub const SOL_USDC_DECIMAL: u8 = 6u8;
+
+// TODO: Confirm the value
+pub const CCM_BYTES_PER_ACCOUNT: usize = 33usize;
+pub const MAX_CCM_BYTES_USDC: usize = 1_000usize;
+pub const MAX_CCM_BYTES_SOL: usize = 1_000usize;
```

Comment:
I'd move these three values to `state-chain/chains/src/sol.rs` as we do for `MAX_SOL_FETCHES_PER_TX` since these are not Solana specific values but rather our own parameters. Not technically true for the last one but let's move it there too because it's not obvious that it's not a parameter of ours.

Reply:
OK. Moved to cf_chains/sols module.